auto_nag/scripts/severity_inconsistency.py

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+
+class SeverityInconsistency(BzCleaner):
+    def description(self):
+        return "Bugs with inconsistent severity flags"
+
+    def has_needinfo(self):
+        return True
+
+    def handle_bug(self, bug, data):
+        self.extra_ni = data
+
+        bugid = str(bug["id"])
+
+        def is_security_flag(flag):
+            return flag.startswith("[access-s")
+
+        flags = bug["whiteboard"].split(", ")
+        whiteboard_severity = next(filter(is_security_flag, flags), None)
+
+        data[bugid] = {
+            "whiteboard_severity": whiteboard_severity,
+            "severity": bug["severity"],
+        }
+
+        return bug
+
+    def get_extra_for_needinfo_template(self):
+        return self.extra_ni
+
+    def columns(self):
+        return ["id", "summary", "severity", "whiteboard_severity"]
+
+    def get_mail_to_auto_ni(self, bug):
+        for f in ["assigned_to", "triage_owner"]:
+            person = bug.get(f, "")
+            if person and not utils.is_no_assignee(person):
+                return {"mail": person, "nickname": bug[f + "_detail"]["nick"]}
```

Comment:
Nit:
```suggestion
                return {"mail": person, "nickname": bug[f"{field}_detail"]["nick"]}
```

f-strings are more pythonic :)